lib/cc/yaml/nodes/sequence.rb

```diff
@@ -113,10 +117,21 @@ def add_value!(value)
 
       protected
 
-        def dup_values
-          @children = @children.map { |child| child.dup }
-          self
+      def dup_values
+        @children = @children.map { |child| child.dup }
+        self
+      end
+
+      private
+
+      def normalize_path_endings(value)
```

Comment:
This seems to be the odd functionality out in this class. What do you think of extracting it to its own `PathNormalizer` class?